.gitignore

```diff
@@ -128,6 +128,10 @@ coverage/
 
 # Generated examples
 packages/rx_bloc_cli/example/test_app
+packages/rx_bloc_cli/example/testapp
+packages/rx_bloc_cli/example/testapp_backup
```

Comment:
Do we really need to have two back-up directories ?

Reply:
I've included the second one (`_bk`) as a precaution in case we need an additional backup during testing (in case we want to analyse multiple versions of a project). This was however from my personal testing and I can remove it.

Reply:
Yes, let's keep a single folder name for the back-up directory. The same as defined in the script

Reply:
Done.